### PR TITLE
[ADD][8.0] l10n_id_taxform_period

### PR DESCRIPTION
**Pull request by andhit-r**
# 
# Indonesia's Taxform - Tax Period

This module add feature to manage Indonesia's taxform period, both
tax year (tahun pajak) and tax period (massa pajak).

This module will be used by other taxform module.
# Usage

To manage taxform period you need to:
1. Go to menu _Taxform -> Configuration -> Period -> Tax Year_
2. Create, edit, or delete tax year data

### CODE REVIEW

**Comment by andhit-r** [comment]
Hi @mikevhe18, @raditv, @azmimr67, @primeforcindo, @m-sani  would you kindly review this PR. Many thanks

File: l10n_id_taxform_period/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/219/8.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA//issues>`_. In case of trouble, please
```
Comment (mikevhe18) [inline l10n_id_taxform_period/README.rst]: https://github.com/OCA/l10n-indonesia/issues

**Review by mikevhe18 — CHANGES_REQUESTED** [review]

**Comment by azmimr67** [comment]
Tested functionally on runboard 👍

**Comment by andhit-r** [comment]
@mikevhe18 i addressed your comment on my last commit. Would you knindly review it it again.

**Comment by mikevhe18** [comment]
:+1: Tested on runbot

**Comment by raditv** [comment]
Finally 👍

**Pull request merged**